test/Scoop-Decompress.Tests.ps1

```diff
@@ -1,62 +1,152 @@
 . "$psscriptroot\Scoop-TestLib.ps1"
-. "$psscriptroot\..\lib\core.ps1"
 . "$psscriptroot\..\lib\decompress.ps1"
 . "$psscriptroot\..\lib\unix.ps1"
+. "$psscriptroot\..\lib\install.ps1"
+. "$psscriptroot\..\lib\manifest.ps1"
+. "$psscriptroot\..\lib\config.ps1"
 
 $isUnix = is_unix
 
-describe "unzip_old" -Tag 'Scoop' {
-    beforeall {
-        $working_dir = setup_working "unzip_old"
+function install_app_ci($app, $architecture) {
+    $manifest = manifest $app
+    $version = $manifest.version
+    $dir = ensure (versiondir $app $version)
+    $fname = dl_urls $app $version $manifest $null $architecture $dir
+    $dir = link_current $dir
+    success "'$app' ($version) was installed successfully!"
+}
+
+function test_extract($extract_fn, $from, $recurse) {
+    $to = (strip_ext $from) -replace '\.tar$', ''
+    & $extract_fn ($from -replace '/', '\') ($to -replace '/', '\') $recurse
+    $to
```

Comment:
??
This will just return `$to` to pipeline. Is this intended?

```suggestion
```

Reply:
Yes, just return `$to`, I left it unchanged as former `test-unzip`, it should be `return $to`